src/components/Transcriber.tsx

```diff
@@ -598,12 +600,24 @@ export function Transcriber(
   }, [paratextIntegration, project, projintegrations]);
 
   useEffect(() => {
-    if (project && project !== '') {
-      var r = findRecord(memory, 'project', project) as Project;
-      if (r) getFontData(r, offline).then((data) => setProjData(data));
+    if (artifactTypeSlug === ArtifactTypeSlug.Vernacular) {
+      if (project) {
+        const r = findRecord(memory, 'project', project) as Project | undefined;
+        if (r) getFontData(r, offline).then((data) => setProjData(data));
+      }
+    } else {
```

Comment:
This should use the project settings if the step doesn't have the settings

Reply:
ok, I'll check for language